src/utils/configure-nuxt.ts

```diff
@@ -54,29 +54,42 @@ export function configureNuxt(
   })
 
   nuxt.hook('prepare:types', ({ references }) => {
-    references.push({ types: 'vuetify' })
+    // references.push({ types: 'vuetify' })
```

Comment:
Testing with and without this, VSCode doesn't seem to work on `LazyVbtn`, IntelliJ works :thinking: